go.mod

```diff
@@ -18,6 +18,7 @@ require (
 	github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90
 	github.com/prometheus/procfs v0.0.5 // indirect
 	github.com/spf13/cobra v0.0.5
+	github.com/spf13/viper v1.3.2
```

Comment:
nit: use a more recent release.

Reply:
done

Reply:
Uhmm using v1.6.1 create some problems with make-verify-gen
Reverting to the older release ATM